package.json

```diff
@@ -18,8 +18,10 @@
     "watch:styles": "watch 'npm run style' ./src/app/styles",
     "publish": "node scripts/server.js",
     "test": "npm run test:unit && npm run test:integration",
-    "test:integration": "cd test && rspec && cd ..",
     "test:unit": "cd test/unit && NODE_PATH=$NODE_PATH:. BABEL_ENV=production mocha setup.js *.spec.js",
+    "test:integration": "cd test && rspec spec/integration_spec.rb && cd ..",
+    "test:proxy": "cd test && ruby spec/sauce/sauce_connect.rb && cd ..",
```

Comment:
This command should be run first and separately from test:browsers (assuming you want Sauce to be able to reach your Docker check-app services), and will tail in the window you run it in. I tried automating this but you probably want to keep it running and then run the test command multiple times, so separate seems better.